Container/Dockerfile

```diff
@@ -74,8 +79,29 @@ RUN make qemu
 RUN make mempool
 RUN make banshee
 
+# Dependencies needed for compiling Snitch
+## Bender's installaton
+RUN wget https://github.com/pulp-platform/bender/releases/download/v${BENDER_VERSION}/bender-${BENDER_VERSION}-x86_64-linux-gnu-ubuntu${UBUNTU_VERSION}.tar.gz && \
+    tar xzf bender-${BENDER_VERSION}-x86_64-linux-gnu-ubuntu${UBUNTU_VERSION}.tar.gz && cp  /app/bender /bin
+ENV PATH=/app/bender:$PATH
+
+## Precompiled llvm 
+RUN latest_tag=`curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/pulp-platform/llvm-project/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'` && \
+echo "SNITCH_LLVM_VERSION=${SNITCH_LLVM_VERSION} LLVM_TAR=${LLVM_TAR} latest_tag=${latest_tag}" && \
+test "${SNITCH_LLVM_VERSION}" = "latest" && SNITCH_LLVM_VERSION=${latest_tag} || : ; \
+LLVM_TAR=riscv32-pulp-llvm-ubuntu2004-$(echo $SNITCH_LLVM_VERSION | cut -d '-' -f3-).tar.gz && \
+mkdir -p riscv-llvm && \
+echo "SNITCH_LLVM_VERSION=${SNITCH_LLVM_VERSION} LLVM_TAR=${LLVM_TAR} latest_tag=${latest_tag}" && \
+wget -qO- https://github.com/pulp-platform/llvm-project/releases/download/${SNITCH_LLVM_VERSION}/${LLVM_TAR} | \
+tar xvz --strip-components=1 -C riscv-llvm
+ENV PATH=/app/riscv-llvm/bin:${PATH}
+
+## compile snitch
+RUN make snitch_runtime
+
 # Remove toolchain to make the container lighter
 RUN rm -rf toolchain
+RUN rm -rf /app/snitch_cluster
```

Comment:
move this at the end of the line 100 like this so that it doesn't create another layer unnecesarily:
```
RUN make snitch_runtime && rm -rf /app/snitch_cluster
```